modules/instrument_builder/jsx/react.questions.js

```diff
@@ -625,16 +625,16 @@ var AddElement = React.createClass({
       if (minDate > maxDate && min !== '' && max !== '') {
         let temp = (this.state.error) ? this.state.error : {};
 
-        temp.dateOption = "End year append befor start year";
+        temp.dateOption = "End year append before start year";
         this.setState({
           error: temp
         });
         hasError = true;
       }
-      if (minYear > 9999 || minYear < 1900 || maxYear > 9999 || maxYear < 1900) {
+      if (minYear > 9999 || minYear < 1000 || maxYear > 9999 || maxYear < 1000) {
         let temp = (this.state.error) ? this.state.error : {};
 
-        temp.dateOption = "The year must have exactly 4 digits. Please choose an integer number between 1900 and 9999.";
+        temp.dateOption = "The year must have exactly 4 digits. Please choose an integer number between 1000 and 9999.";
```

Comment:
I would have prefer to keep the minimum year at 1900 and adjust the message like : 
"The year must have exactly 4 digits and be greater than 1900. Please choose an integer number between 1900 and 9999."

Reply:
y 1900 ? we can clearly have values in the 1800's or 1700's no ?

Reply:
@ridz1208 the oldest living human being was born in 1901. technically you are correct but I don't see why we would allow it.

Reply:
@ridz1208 see comment from @driusan hidden somewhere in these comment.

Reply:
You'd have to be over 118 years old to be born in 1900 which is more than the oldest (known) person on earth.  There is about zero chance that a date for a visit, other test or date of birth would be lower than that.

Reply:
what about the date of your grand father's birth ? or the date your house was built? 

there is no boundary to questions that would come up in an instrument... its not only age

Reply:
@ridz1208 I only reported what  @driusan comments. For the records, my grandparent where born in early 20th century and I only know the year for 2 of them. Yes, date the house was build may be valid, specially on the other side of the pound.